style.css

```
@@ -0,0 +1,76 @@
+body {
+  background-image: url("/home/wihogora/Brenda_portfolio/images/backgrd.svg");
+  background-size: auto;
+}
```

```suggestion
body {
  background-image: url("./images/backgrd.svg");
  background-size: auto;
  background-repeat: no-repeat;
}
```

- [ ] Kindly add the correct path to the image and a background no repeat as suggested above ⬆️ 
- [ ] Kindly add the background image to the right section in your case the "contcontainer" secton